src/adapter/src/catalog/objects.rs

```diff
@@ -1821,6 +1849,10 @@ impl mz_sql::catalog::CatalogItem for CatalogEntry {
         self.name()
     }
 
+    fn has_columns(&self) -> bool {
+        self.has_columns()
```

Comment:
uncovered but just calls the other function so looks harmless. Why does it have to exist at all?

Reply:
Removed it!